.github/workflows/ci.yaml

```diff
@@ -58,3 +58,35 @@ jobs:
           components: rustfmt
           override: true
       - run: cargo fmt -- --check
+
+  coverage:
+    name: "Coverage: nightly"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+          components: llvm-tools-preview
+          override: true
+      - run: cargo install grcov --force --locked
+      - run: |
+          cargo test --all-features --all-targets
+          cargo run --example decode_file
+          cargo run --example decode
```

Comment:
I don't think examples should contribute to coverage.

Reply:
Ok, I removed the examples from the coverage runs and output - so they just won't be recorded at all and will also not drag down the coverage